simvue/executor.py

```diff
@@ -131,6 +143,12 @@ def _exec_process(
 
             exit_status_dict[proc_id] = _result.returncode
 
+            run_on_exit(
+                status_code=exit_status_dict[proc_id],
+                std_out=std_out[proc_id],
```

Comment:
What happens if the user calls the same process with the same name more than once - maybe raise an issue for this